### PR TITLE
C2C-333: Added lab test panels for the CRP and ASO.

### PR DESCRIPTION
**Pull request by kazlaw**
Link to ticket: https://mekomsolutions.atlassian.net/browse/C2C-333

[Screencast from 23-08-24 15:20:57.webm](https://github.com/user-attachments/assets/ad00b90f-75ba-4d6d-9225-c39f938834b9)

### CODE REVIEW

**Comment by kazlaw** [comment]
> Do we need to make corresponding changes on the Odoo Config @kazlaw?

@Ruhanga The configs current work without the configs in Odoo. Could this be a problem?

File: configuration/concepts/labs.csv [hunk 1]
```diff
@@ -9,8 +9,10 @@ d3fe4806-82e3-4e7e-a37c-8deaf770b6dc,,,Orange,Jaune Orange,Orange,Jaune Orange,,
 a9d907d9-92d2-4453-8dc5-d32bf7f25324,,,Anti-strepyolysine O (coded):,Anti-strepyolysine O (coded):,Anti-strepyolysine O (coded):,Anti-strepyolysine O (coded):,,,LabTest,Coded,,2dc241e2-a3e2-465d-8876-f781b5316708;2225d327-c287-471a-b03f-0f4a809a1a18,,,,,,,,,
 ce0f626b-0cc3-463f-b703-e4b7c5e60f90,,CIEL:161500,C Reactive Protein,C Reactive Protein,CRP,CRP,,,LabTest,Numeric,,,,,,,,,,,
 ee37c9bd-ee01-4292-b1ff-ac5ee62ebe80,,,C Reactive Protein (coded):,C Reactive Protein (coded):,C Reactive Protein (coded):,C Reactive Protein (coded):,,,LabTest,Coded,,2dc241e2-a3e2-465d-8876-f781b5316708;2225d327-c287-471a-b03f-0f4a809a1a18,,,,,,,,,
+02bfdeaf-9327-4efe-8cd6-040d0326b0c2,,,Anti-strepyolysine O Test,Anti-strepyolysine O Test,Anti-strepyolysine O Test,Anti-strepyolysine O Test,,,LabSet,N/A,,,ASO;Anti-strepyolysine O (coded):,,,,,,,,
+86d9fb4a-84f7-4c0e-a74c-c3012742bb24,,,C Reactive Protein Test,C Reactive Protein Test,C Reactive Protein Test,C Reactive Protein Test,,,LabSet,N/A,,,CRP;C Reactive Protein (coded):,,,,,,,,
```
Comment (Ruhanga) [inline configuration/concepts/labs.csv:13]: Should this not be:
```suggestion
86d9fb4a-84f7-4c0e-a74c-c3012742bb24,,,C Reactive Protein Test,C Reactive Protein Test,C Reactive Protein Test,C Reactive Protein Test,,,LabSet,N/A,,,C Reactive Protein;C Reactive Protein (coded):,,,,,,,,

```

File: configuration/concepts/labs.csv [hunk 2]
```diff
@@ -9,8 +9,10 @@ d3fe4806-82e3-4e7e-a37c-8deaf770b6dc,,,Orange,Jaune Orange,Orange,Jaune Orange,,
 a9d907d9-92d2-4453-8dc5-d32bf7f25324,,,Anti-strepyolysine O (coded):,Anti-strepyolysine O (coded):,Anti-strepyolysine O (coded):,Anti-strepyolysine O (coded):,,,LabTest,Coded,,2dc241e2-a3e2-465d-8876-f781b5316708;2225d327-c287-471a-b03f-0f4a809a1a18,,,,,,,,,
 ce0f626b-0cc3-463f-b703-e4b7c5e60f90,,CIEL:161500,C Reactive Protein,C Reactive Protein,CRP,CRP,,,LabTest,Numeric,,,,,,,,,,,
 ee37c9bd-ee01-4292-b1ff-ac5ee62ebe80,,,C Reactive Protein (coded):,C Reactive Protein (coded):,C Reactive Protein (coded):,C Reactive Protein (coded):,,,LabTest,Coded,,2dc241e2-a3e2-465d-8876-f781b5316708;2225d327-c287-471a-b03f-0f4a809a1a18,,,,,,,,,
+02bfdeaf-9327-4efe-8cd6-040d0326b0c2,,,Anti-strepyolysine O Test,Anti-strepyolysine O Test,Anti-strepyolysine O Test,Anti-strepyolysine O Test,,,LabSet,N/A,,,ASO;Anti-strepyolysine O (coded):,,,,,,,,
```
Comment (Ruhanga) [inline configuration/concepts/labs.csv:12]: And this:
```suggestion
02bfdeaf-9327-4efe-8cd6-040d0326b0c2,,,Anti-strepyolysine O Test,Anti-strepyolysine O Test,Anti-strepyolysine O Test,Anti-strepyolysine O Test,,,LabSet,N/A,,,Anti-strepyolysine O;Anti-strepyolysine O (coded):,,,,,,,,

```

**Review by Ruhanga — CHANGES_REQUESTED** [review]
Strange that it's working without adding corresponding config changes in Odoo. However there a few more suggestions below.

**Comment by Ruhanga** [comment]
> @Ruhanga The configs current work without the configs in Odoo. Could this be a problem?

Ideally, we need to perform a similar task in this [Odoo config](https://github.com/mekomsolutions/odoo-config-c2c/blob/1be249469393ba77dab10d64ec09c38742c2d116/configuration/product_variant/labs.csv), effectively merging the existing `Anti-strepyolysine O`and `Anti-strepyolysine O (coded)` into `Anti-strepyolysine O Test` and doing the same for  `C Reactive Protein`.

**Review by Ruhanga — APPROVED** [review]
LGTM

**Pull request merged**